_posts/2019-09-30-Home-automation-part-1.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+authors: [kevin_van_den_abeele]
+title: "Home Automation part 1"
+image: /img/2019-09-30-home-automation/banner.jpg
+tags: [Internet of Things, Smart tech, Home, smart home, automation, home automation, siri, apple, google assistant, amazon, alexa, homekit, home assistant, openHab]
+category: Machine Learning
+comments: true
+---
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/css/lightbox.css" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/bootstrap.css" />
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/js/lightbox.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/index.min.js"></script>
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Platforms](#platforms)
+3. [Example](#example)
+4. [Conclusion](#conclusion)
+5. [Resources](#resources)
+
+## Introduction
+
+In this series of blogposts we are going to take a deeper look into home automation.
+Home automation is the wonderful art of automating your home so it becomes easier to do certain tasks.
+It is a very wide field where there are many available options, platforms and devices.
+If you are willing to get creative almost everything can be automated!
+
+In this first blogpost we'll present some of the platforms that are available and some of the pros & cons of each one.
+To close of this blogpost we'll start by looking in to Apple's Homekit and by extension HomeBridge.
+
+## Platforms
+
+In the world of home automation there are many ways to automate stuff.
+We will be focussing on either cloud-enabled platforms or those that you can run on premise.
+The hardware vender based platforms (e.g.: niko) will not be included!
+
+The platforms can be split up into two main parts:
+
+- Cloud enabled, mainly from manufactures and larger phone companies
+- On premise platforms (that do not require internet/cloud access)
+
+Security is a very important aspect in the home automation world!
+The best way one would go about to automating one's home would be completely independent from any cloud based services/APIs
+Internet access can be intermittent, cloud based API's can be down or event be shut down permanently.
+Even worse, accounts could be breached giving unknown third parties access to your home's devices!
+
+However, most people already have some smart devices and most of these have their own app, so ditching any cloud based systems is going to be a lot harder/more expensive.
+As with any online service please, for the love of god, use strong unique password for each different service and enable multi factor authentication wherever possible!
```

Comment:
Ik zou hier please, for the love of god weglaten 😅

"As with any online service, use **a** strong unique password for each separate service and enable **multifactor** authentication **whenever** possible!"